microsoft-edge/web-platform/release-notes/135.md

```diff
@@ -0,0 +1,462 @@
+    * [highlightsFromPoint API](#highlightsfrompoint-api)
+  * [Deprecated and removed features](#deprecated-and-removed-features)
+    * [Deprecate `Intl.Locale` getters](#deprecate-intllocale-getters)
+    * [Remove `navigator.xr.supportsSession` method](#remove-navigatorxrsupportssession-method)
+    * [Remove WebGPU limit `maxInterStageShaderComponents`](#remove-webgpu-limit-maxinterstageshadercomponents)
+* [Origin trials](#origin-trials)
+  * [Microsoft Edge-only origin trials](#microsoft-edge-only-origin-trials)
+    * [Digital Goods API](#digital-goods-api)
+    * [Web App LocalFolder Access](#web-app-localfolder-access)
+    * [HTML+IDL handwriting attribute](#htmlidl-handwriting-attribute)
+    * [Acquisition Info API](#acquisition-info-api)
+
+
+<!-- ====================================================================== -->
+## Edge DevTools
+
+For Edge DevTools, see [What's New in Microsoft Edge DevTools](../../devtools-guide-chromium/whats-new/whats-new.md).
+<!-- todo: when exists in April 2025, use instead:
+For Edge DevTools, see [What's New in DevTools (Microsoft Edge 135)](../../devtools-guide-chromium/whats-new/2025/04/devtools-135.md).
+-->
+
+
+<!-- ====================================================================== -->
+## WebView2
+
+For WebView2, see [Release Notes for the WebView2 SDK](../../webview2/release-notes/index.md).
+<!-- todo: when exists in April 2025, use instead:
+For WebView2, see [1.0.n.n](../../webview2/release-notes/index.md#10nnnnnn) in _Release Notes for the WebView2 SDK_ (Apr. 2025).
+-->
+
+
+<!-- ====================================================================== -->
+## Web platform features
+
+
+<!-- ------------------------------ -->
+#### CSS features
+
+
+<!-- ---------- -->
+###### CSS carousel widget: `::column` pseudo-element
+
+The `::column` CSS pseudo-element represents the individual columns in a multi-column container.
+
+The container can be styled with a limited amount of CSS properties.
+
+The container can be snapped to by its scrolling container, by using `::scroll-marker` pseudo-elements.
+
+
+<!-- ---------- -->
+###### CSS carousel widget: `::scroll-marker` and `::scroll-marker-groups` pseudo-elements
+
+The `::scroll-marker` and `::scroll-markers-group` pseudo-elements allow creating a set of focusable markers within a scroll container.
+
+The set of focusable markers is used to snap the scroll container to specific positions.
+
+
+<!-- ---------- -->
+###### CSS carousel widget: `::scroll-button()` pseudo-element
+
+The `::scroll-button()` pseudo-element allows creating scroll buttons as pseudo-elements of a scroll container.
+
+The scroll buttons are used to scroll the container in the direction of the button, when the scroll buttons are clicked.
+
+
+<!-- ---------- -->
+###### Nested pseudo-elements styling
+
+Pseudo-elements that are nested inside other pseudo-elements can now be styled, by using `::before::marker` and `::after::marker`.
```

Comment:
```suggestion
Pseudo-elements that are nested inside other pseudo-elements can now be styled, by using `::before::marker` and `::after::marker`.

`::column::scroll-marker` is not yet supported.
```
Shall we specify the item of interest that's not supported yet?  This suggested wording is better than promising "will be added".

orig draft:
Pseudo-elements that are nested inside other pseudo-elements can now be styled. So far, `::before::marker` and `::after::marker` are supported. Support for `::column::scroll-marker` will be added in the future.

Reply:
Yes, I believe we need to add the bit about `::column::scroll-marker`.